### PR TITLE
Fc 564 initiate save

### PR DESCRIPTION
**Pull request by leewadhams**

### CODE REVIEW

File: src/Web/Sfa.Das.Sas.Shared.Components/ViewComponents/SaveApprenticeship/SaveApprenticeshipViewComponent.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        private readonly IFatConfigurationSettings _config;
+
+        public SaveApprenticeshipViewComponent(IFatConfigurationSettings config)
+        {
+            _config = config;
+        }
+
+        public IViewComponentResult Invoke(string apprenticeshipId)
+        {
+            var saveUrl = new Uri(new Uri(_config.EmployerFavouritesUrl), "/save-apprenticeship-favourites");
```
Comment (desderry) [inline src/Web/Sfa.Das.Sas.Shared.Components/ViewComponents/SaveApprenticeship/SaveApprenticeshipViewComponent.cs]: should the save url be part of config along with the url?

File: src/Web/Sfa.Das.Sas.Shared.Components/ViewComponents/SaveApprenticeship/SaveApprenticeshipViewComponent.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            var builder = new UriBuilder(saveUrl);
```
Comment (desderry) [inline src/Web/Sfa.Das.Sas.Shared.Components/ViewComponents/SaveApprenticeship/SaveApprenticeshipViewComponent.cs]: Should this logic reside in an orchestrator? what if a consumer of the shared package doesnt use the view component but needs the save url?

File: src/Web/Sfa.Das.Sas.Shared.Components/ViewComponents/SaveApprenticeship/SaveApprenticeshipViewComponent.cs (hunk 1 above)
Comment (leewadhams) [reply src/Web/Sfa.Das.Sas.Shared.Components/ViewComponents/SaveApprenticeship/SaveApprenticeshipViewComponent.cs]: Yeah I was toying with this one. Will move it to the config.

File: src/Web/Sfa.Das.Sas.Shared.Components/ViewComponents/SaveApprenticeship/SaveApprenticeshipViewComponent.cs (hunk 2 above)
Comment (leewadhams) [reply src/Web/Sfa.Das.Sas.Shared.Components/ViewComponents/SaveApprenticeship/SaveApprenticeshipViewComponent.cs]: Have pulled the logic out into separate class.

**Pull request merged**